src/pages/_app.tsx

```diff
@@ -62,6 +64,28 @@ export default function App({ Component, pageProps }: AppProps) {
       axiosInstance.defaults.headers.common['auth'] = `${token}`
     }
   }, [])
+  const [smallscreen, setSmallScreen] = useState(false)
+  const width = useWindowWidth()
+
+  useEffect(() => {
+    if (width < 1024) {
```

Comment:
this can be replaced by :

setSmallScreen(width < 1024)

Reply:
@jainkrati 

It sure can. I like the more expressive form, using `if` though. It is much easier to read and understand.

Reply:
Here is why concise code is important: https://debugpointer.com/code/write-concise-code, also pls refer to point 2 as it talks about if else condition